### DISCUSSION TITLE
Best practice to launch a Dioxus app with axum ?

### DISCUSSION THREAD

**Discussion by XavDesbordes** [DioxusLabs/dioxus#2290] [Q&A]
Hi,
I was crawling in the example section and many examples require `dx serve` to be launched.
Is there an example I missed that work with axum directly, targeting a production release ?
I would appreciate to see how it is articulated.

**Comment by XavDesbordes** [comment]
is the combo 
```
dx build --features web --release
cargo run --features server --release // or ./app later on
```
all it takes ?
Talking for v0.5.1.

Thanks

**Comment by XavDesbordes** [comment]
Yes, except I need to tailor things:
[https://github.com/DioxusLabs/dioxus/tree/main/packages/fullstack/examples](https://github.com/DioxusLabs/dioxus/tree/main/packages/fullstack/examples)

**Comment by XavDesbordes** [comment]
I will reopen because I would like a confirmation that this way of doing things is ok for production.

**Comment by ealmloff** [comment]
You can just do:
```
dx build --release
```
and then run the executable it produces in the `./dist` directory for the server
```
./dist/myproject
```

**Reply by XavDesbordes** [reply]
Yes, works perfectly.
As for TLS, I will dig into the existing examples.

Thanks ealmloff!
